services/motion/builtin/builtin_test.go

```diff
@@ -509,93 +509,6 @@ func TestObstacleReplanningGlobe(t *testing.T) {
 	}
 }
 
-func TestObstacleReplanningSlam(t *testing.T) {
```

Comment:
yea, not super satisfying.. T^T

I will say that what we test in `TestObstacleReplanningSlam` is also tested in `TestObstacleReplanningGlobe`.
Further, `TestObstacleReplanningSlam` isn't really a unit test, rather it is an integration test.. so I am ok with removing it.

Reply:
I talked to Miko about this.  It feels like this test in particular is a little duplicative of another test we have exercising replanning for MoveOnGlobe.  Since it seems at the moment that the flake is not an error in business logic and rather a result of the difficulty of writing integration tests, I didnt think it was super necessary to spend a lot of time on this when we are already testing the components it tests elsewhere.